### PR TITLE
Exercise 1

### PR DESCRIPTION
**Pull request by samyou-softwire**

### CODE REVIEW

File: todo_app/app.py [hunk 1]
```diff
@@ -8,4 +9,10 @@
...
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template("index.html", items=get_items())
+
+
+@app.route('/add', methods=['POST'])
+def add():
+    add_item(request.form.get("name"))
+    return redirect(url_for("index"))
```
Comment (CharlieCumber1) [inline todo_app/app.py:18]: Great! - We should use `url_for` as much as possible when referencing endpoints as it makes the code much easier to maintain.

File: todo_app/templates/index.html [hunk 2]
```diff
@@ -2,17 +2,27 @@
...
-      </ul>
+    <div class="row">
+        <div class="col-md-12">
+            <h2>Items</h2>
+            <ul class="list-group mb-4">
+              {% for item in items %}
+                <li>{{ item.title }}</li>
+              {% endfor %}
+            </ul>
+            <form id="addform" action="{{ url_for("add") }}" method="post">
```
Comment (CharlieCumber1) [inline todo_app/templates/index.html:18]: Great!

File: todo_app/templates/index.html [hunk 3]
```diff
@@ -2,17 +2,27 @@
...
+                <h2>Add a new one</h2>
+                <label for="itemname">Name</label>
+                <input type="text" id="itemname" name="name">
```
Comment (CharlieCumber1) [inline todo_app/templates/index.html:21]: You are calling the field `name` when there is already a pattern of calling it `title` as per line 15.

Can you change the field to be named `title`?

**Review by CharlieCumber1 — COMMENTED** [review]
Great work - just a small change required before you can merge.

File: todo_app/templates/index.html (hunk 3 above)
Comment (samyou-softwire) [reply todo_app/templates/index.html:21]: Good point - will change now

**Comment by CharlieCumber1** [comment]
Feel free to merge your branch to master

**Pull request merged**